src/_data/sidenav.yml

```diff
@@ -36,7 +36,6 @@
       permalink: /docs/codelabs
 
 - title: Development
-  permalink: /docs/development
```

Comment:
This shouldn't have been removed. It will break breadcrumbs, among other things.